projects/assets/src/translations/translation-chunks-config.ts

```diff
@@ -73,17 +73,20 @@ export const translationChunksConfig: TranslationChunksConfig = {
     'register',
     'checkoutLogin',
   ],
+
   organization: [
+    // 'costCenterBudget',
+    // 'costCenter',
+    // 'costCenterList',
+    // 'costCenterDetails',
+    // 'costCenterForm',
+    // 'costCenterBudgets',
+    // 'costCenterAssignBudgets',
+
     'breadcrumbs',
```

Comment:
`breadcrumbs.XXX` key sounds quite general. Why are do we place it in the `organization.json` chunk?

Reply:
I guess we are moving away from it, as Tobias explained the breadcrumb mechanism was not well designed. 
So all of it will be ideally removed like costCenter stuff.

Reply:
Organisation in this chunk will varnish when we move all to feature lib. And breadcrumbs should not be driven by localisations as much as possible.